### ISSUE TITLE
Font styling

### ISSUE THREAD

**Issue by s4mstruthers** [GiPHouse/DPD-Drawing-Tool#48]
As an end-user, I want to read the text without any difficulty, to enhance accessibility. The font style should be clear and simple and the text should have a proper size.
